Sources/Tokenizers/BPETokenizer.swift

```diff
@@ -77,8 +75,6 @@ class BPETokenizer: PreTrainedTokenizerModel {
 
         bosToken = tokenizerConfig.bosToken?.stringValue
         bosTokenId = bosToken == nil ? nil : tokensToIds[bosToken! as NSString]
-
-        fuseUnknownTokens = tokenizerConfig.fuseUnk?.boolValue ?? false
```

Comment:
Same, these lines were part of #117 (I started that thread from the branch of #113, that may have caused some confusion, sorry!)